service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseValidatorServiceTest.java

```diff
@@ -301,10 +354,63 @@ private Allocation allocationProposal() {
             .build());
     }
 
+    private List<Element<Applicant>> applicantWithInvalidEmailAddress(final String emailAddress) {
+        return wrapElements(Applicant.builder()
+            .leadApplicantIndicator("Yes")
+            .party(ApplicantParty.builder()
+                .organisationName("Harry Kane")
+                .jobTitle("Judge")
+                .address(Address.builder()
+                    .addressLine1("1 Some street")
+                    .addressLine2("Some road")
+                    .postTown("some town")
+                    .postcode("BT66 7RR")
+                    .county("Some county")
+                    .country("UK")
+                    .build())
+                .email(EmailAddress.builder()
+                    .email(emailAddress)
+                    .build())
+                .telephoneNumber(Telephone.builder()
+                    .telephoneNumber("02838882404")
+                    .contactDirection("Harry Kane")
+                    .build())
```

Comment:
Considering cutting some properties. All we are interested in is the email for validation.

Reply:
updated